dashboard.el

```diff
@@ -54,6 +55,7 @@ Return entire list if `END' is omitted."
   :syntax-table nil
   :abbrev-table nil
   (linum-mode -1)
+  (page-break-lines-mode 1)
```

Comment:
We're already activating page-break-lines-mode here:
https://github.com/rakanalh/emacs-dashboard/pull/20/files#diff-49993283b3a3adc31101f309fab24b1cR265